libraries/ms-common/src/can.c

```diff
@@ -145,14 +152,15 @@ bool can_process_event(const Event *e) {
 }
 
 void prv_tx_handler(void *context) {
-  CanStorage *can_storage = context;
-  CanMessage tx_msg;
-
-  // If we failed to TX some messages or aren't transmitting fast enough, those
-  // events were discarded. Raise a TX event to trigger a transmit attempt. We
-  // only raise one event since TX ready interrupts are 1-to-1.
-  if (can_fifo_size(&can_storage->tx_fifo) > 0) {
-    event_raise(can_storage->tx_event, 0);
+  if (TX_CALLBACK_ENABLE) {
```

Comment:
Like previous comment, please explain here what the bug is, including the ticket number, and why this fixes it.